cterasdk/client/http.py

```diff
@@ -143,6 +143,14 @@ def get_session_id(self):
     def set_session_id(self, session_id):
         self.session.cookies.set(self._session_id_key, session_id)
 
+    def set_custom_headers(self, headers):
```

Comment:
Won't your headers get compounded? Consider the following:
1. Session headers is set to X -> All consequent commands will include the X header
2. Header Y is added for a certain command -> All consequent commands will include both X and Y headers
3. Header Z is added for a certain command -> All consequent commands will include all three headers
In my experience at some point this will generate an error

Reply:
Yes. Headers are compounded. I updated the method documentation to reflect that better. If needed, we can change the implementation to keep the custom headers in a class variable and then pass them in the individual HTTP request calls (i.e. merging them with other headers)